tasks.mak

```diff
@@ -73,6 +73,9 @@ test_metadata_update:
 test_metadata_validate:
 	go test ./integration_test/validate_metadata_test.go
 
+test_set:
+	go test ./internal/set
```

Comment:
this seems like an oddly specific thing to have in the makefile. How often is this going to get used after the PR is merged? I'd say likely never.

Reply:
I just kind of got in the habit of adding a target for each package, but I don't have to keep doing that.

Reply:
ok

Reply:
Just use a pattern rule to define tests on arbitrary packages.

Reply:
Yeah I suppose that would work too. I'll remove this target from this PR and do that in a future follow up

Reply:
Done in [8c8978f](https://github.com/GoogleCloudPlatform/ops-agent/pull/901/commits/8c8978ffc91e2e1e2dcc118c3017717a6d266f39)